polar-core/src/polar.lalrpop

```diff
@@ -478,15 +478,15 @@ OrExp<T>: Value = {
     },
 }
 
-Exp2<T>: ValueOrLogical = {
+Exp1<T>: ValueOrLogical = {
```

Comment:
Can we swap this & `OrExp` w/ `Exp1` & `AndExp` so this file remains in precedence order after the change?